docs/markdown/Pkgconfig-module.md

```diff
@@ -44,7 +44,9 @@ keyword arguments.
   `${PREFIX}/include/foobar-1`, the correct value for this argument
   would be `foobar-1`
 - `requires` list of strings, pkgconfig-dependencies or libraries that
-   `pkgconfig.generate()` was used on to put in the `Requires` field
+  `pkgconfig.generate()` was used on to put in the `Requires` field.
+  *Since 1.9.0* internal dependencies are supported if `pkgconfig.generate()`
+  was used on them.
```

Comment:
Could this be rewritten to make it more clear? I don't understand what it is trying to say.